test/core/test_serialization.py

```diff
@@ -1,8 +1,11 @@
 import io
 import pickle
 
+from mathics.core.load_builtin import import_and_load_builtins
 from mathics.session import MathicsSession
 
+import_and_load_builtins()
```

Comment:
I had to add this line on each test that I want to run individually. Maybe it should be added in `mathics.session`.